boltrouter/bolt_request.go

```diff
@@ -188,7 +188,7 @@ func (br *BoltRouter) DoBoltRequest(logger *zap.Logger, boltReq *BoltRequest) (*
 			return resp, false, err
 		} else if !StatusCodeIs2xx(resp.StatusCode) && resp.StatusCode == 404 {
 			// if the request to AWS failed with 404: NoSuchKey, fall back to Bolt
-			logger.Warn("aws request failed, falling back to bolt", zap.Int("statusCode", resp.StatusCode))
+			logger.Error("aws request failed, falling back to bolt", zap.Int("statusCode", resp.StatusCode))
```

Comment:
same here, is error really the right call?

Reply:
Ditto as above.